doc/s9s-cluster.1

```diff
@@ -427,6 +427,20 @@ s9s cluster \\
     --wait
 .fi
 
+The \fB\-\-unregister-only\fR option removes the node from the cluster's node list without
+stopping or removing any running processes. If both \fB\-\-uninstall\fR and \fB\-\-unregister-only\fR
```

Comment:
I think the important thing here will be whether the node replication connections to the rest of the cluster nodes are kept, or dropped.
If dropped, then this is a remove node without uninstall.
Lets improve the documentation to be clear, that the db node will still be part of the db cluster, just not managed by CC.

Further more, I can imagine that we gonna have problems about this. The still managed nodes will reference to this not managed node, and this unregistered node will not be updated in case of failover done by cmon.

If failover or promotion to the unregistered node happens outside of CC, then CC will not see the master and that will lead to further bugs.

So this feature was not added before, because the consequences are potentially very error prone.

Reply:
I have checked cmon code. We indeed alredy have this feature, despite it is very error prone.
Okay, lets go on. But please add some info here in the doc, that the removed node will still be part of the db cluster, its configuration will not be changed.